src/options.ts

```diff
@@ -115,6 +115,7 @@ export function normalizeOptions(cliArgs: CliArgs): RunOptions {
   options.locations = cliArgs.locations ?? monitor?.locations;
   options.privateLocations =
     cliArgs.privateLocations ?? monitor?.privateLocations;
+  options.tags = cliArgs.tags ?? monitor?.tags;
```

Comment:
@vigneshshanmugam @andrewvc 

I have a question:

This block starts with:

 /**
   * Get the default monitor config from synthetics.config.ts file
  */

But I've noticed that we are just adding:

- schedule
- locations
- privateLocations
- tags (added in this PR)

If you have a look at https://github.com/elastic/synthetics/blob/main/src/dsl/monitor.ts#L47

you will see there are options that we are not getting there. Is there a reason for that? Probably I'm missing context, too. but I believe it's important to ask, just in case